### DISCUSSION TITLE
ISRC for now_playing?

### DISCUSSION THREAD

**Discussion by PiggyPlex** [AzuraCast/AzuraCast#4950] [Q&A] [answered]
[ @SlvrEagle23 ]
If songs have metadata set correctly, is it possible to get the ISRC value of a song for the now_playing endpoint?
Thanks!

**Comment by BusterNeece** [comment, marked as answer]
Answered via Discord, but the answer is to create a new Custom Field for ISRC, and have it automatically set by the ISRC field in the file's metadata. This can be done for as many fields as desired.
